libtransmission/platform-quota.c

```diff
@@ -284,7 +284,7 @@ getquota (const char * device)
       spaceused = (int64_t) dq.dqb_curblocks >> 1;
 #elif defined(__APPLE__)
       spaceused = (int64_t) dq.dqb_curbytes;
-#elif defined(__UCLIBC__)
+#elif defined(__UCLIBC__) && (__UCLIBC_MAJOR__ == 0 || (__UCLIBC_MAJOR__ == 1 && __UCLIBC_SUBLEVEL__ <= 17))
```

Comment:
Actually, you need to use `TR_UCLIBC_CHECK_VERSION` macro from utils.h, otherwise we'll have issues with versions like 1.1.17, 1.2.17, etc.